v20.2/show-schedules.md

```diff
@@ -0,0 +1,129 @@
+
+## Response
+
+The output of `SHOW SCHEDULES` is sorted by creation time. The following fields are returned for each schedule:
+
+Field | Description
+------+------------
+`id` | A unique ID to identify each schedule. This value is used if you want to control schedules (i.e., [pause](pause-schedules.html), [resume](resume-schedules.html), or [drop](drop-schedules.html) it).
+`label` | The name used to identify the backup schedule, given at the time of schedule creation.
+`schedule_status` | The schedule's current status.
+`next_run`  | The [`TIMESTAMP`](timestamp.html) at which the next backup in the schedule is slated to run.
+`state` | Displays last known errors or messages about the backup schedule.
+`recurrence` | How often the backup is taken, which is set at the time of schedule creation.
+`jobsrunning` | The number of [jobs](show-jobs.html) currentl running for the schedule.
+`owner` | The [user](create-user.html) who created the backup schedule. Users with active schedules cannot be dropped.
+`created` | The [`TIMESTAMP`](timestamp.html) when the job was created.
+`command` | The [`CREATE SCHEDULE FOR BACKUP`](create-schedule-for-backup.html) statement used
+
+## Examples
+
+### Show schedules
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW SCHEDULES;
+~~~
+
+~~~
+          id         |        label        | schedule_status |         next_run          |                                                                                                     state                                                                                                     | recurrence | jobsrunning | owner |             created              |                                                                                                                     command
+---------------------+---------------------+-----------------+---------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------+-------------+-------+----------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  588796190000218113 | schedule_name       | ACTIVE          | 2020-09-15 00:00:00+00:00 | reschedule: backup job 589962988741296130 failed with err=backup job 589962988741296130 scheduled by 588796190000218113 failed with status failed                                                             | @daily     |           0 | root  | 2020-09-10 16:52:16.846944+00:00 | {"backup_statement": "BACKUP INTO LATEST IN 's3://test/schedule-test2?AWS_ACCESS_KEY_ID=x&AWS_SECRET_ACCESS_KEY=redacted' WITH revision_history, detached", "backup_type": 1}
```

Comment:
do you think we should clear out "failed with err=..." part? Pretty sure it's a kink/bug in the current implementation.